### PR TITLE
Master warning partner reconciliation nby

### PR DESCRIPTION
**Pull request by Goaman**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by Goaman** [comment]
The fix commit is there to be able to test the reconciliation in the interface.
It has to be removed before merging.

File: addons/account/models/company.py [hunk 1]
```diff
@@ -101,6 +101,10 @@ class ResCompany(models.Model):
     account_invoice_onboarding_state = fields.Selection([('not_done', "Not done"), ('just_done', "Just done"), ('done', "Done"), ('closed', "Closed")], string="State of the account invoice onboarding panel", default='not_done')
     account_dashboard_onboarding_state = fields.Selection([('not_done', "Not done"), ('just_done', "Just done"), ('done', "Done"), ('closed', "Closed")], string="State of the account dashboard onboarding panel", default='not_done')
     invoice_terms = fields.Text(string='Default Terms and Conditions', translate=True)
+    default_rec_partner_journal_id = fields.Many2one('account.journal',
```
Comment (Goaman) [inline addons/account/models/company.py]: account_partner_missmatch_journal_Id

File: addons/account/static/src/js/reconciliation/reconciliation_action.js [hunk 2]
```diff
@@ -213,6 +213,7 @@ var StatementAction = AbstractAction.extend({
         var linesToDisplay = this.model.getStatementLines();
         var linePromises = [];
         _.each(linesToDisplay, function (line, handle) {
+            line.multiPartnerJournal = self.model.multiPartnerJournal
```
Comment (Goaman) [inline addons/account/static/src/js/reconciliation/reconciliation_action.js]: same name as the field in the company

File: addons/account/models/account_move.py [hunk 3]
```diff
@@ -157,15 +157,11 @@ def _onchange_date(self):
     reconcile_model_id = fields.Many2many('account.reconcile.model', compute='_compute_reconcile_model', search='_search_reconcile_model', string="Reconciliation Model", readonly=True)
     # Dummy Account field to search on account.move by account_id
     dummy_account_id = fields.Many2one('account.account', related='line_ids.account_id', string='Account', store=False, readonly=True)
-    tax_cash_basis_rec_id = fields.Many2one(
```
Comment (Goaman) [inline addons/account/models/account_move.py]: check the tax_cash_basis is still working

File: addons/account/models/reconciliation_widget.py [hunk 4]
```diff
@@ -755,6 +759,89 @@ def _get_move_line_reconciliation_proposition(self, account_id, partner_id=None)
             return Account_move_line.browse(pairs[0])
         return Account_move_line
 
+    @api.model
+    def _transfer_partners_lines_to_moves(self, account_move_lines):
```
Comment (Goaman) [inline addons/account/models/reconciliation_widget.py]: check the bug with the same partner

**Comment by pedrobaeza** [comment]
I have come to this through codetriage.com. This seems something abandoned or stalled. Should be closed, @qdp-odoo ?

**Comment by qdp-odoo** [comment]
not yet: the internal task hasn't been cancelled: it has been delayed because of the prioritization made for v13

**Comment by qdp-odoo** [comment]
I suppose it _now_ can be closed... :see_no_evil:

**Pull request closed without merging**